src/AppBundle/Controller/LandingController.php

```diff
@@ -50,11 +50,19 @@ public function contactsAction()
     {
         $contacts = $this->getContactRepository()->findAll();
         shuffle($contacts);
+        $cities = $this->getCityRepository()->findAll();
+        $citiesWithContacts = array_filter(
+            $cities,
```

Comment:
@daria-sieroshtan 
You can do this inline:
```PHP
        $citiesWithContacts = array_filter(
            $this->getCityRepository()->findAll(),
            function ($city) {
                return count($city->getContacts()) > 0;
            }
        );
```
Yet better you can implement method in repo which will return only cities with contacts by doing `->innerJoin('city.contacts')` in QueryBuilder
Everything that can be done by DB should be done by DB. All DB calls should reside in repo